client/citydlg.h

```diff
@@ -298,6 +304,8 @@ class city_dialog : public QWidget {
   void update_building();
   void update_info_label();
   void update_buy_button();
+  void fill_citizens_pixmap(QPixmap *pixmap, QPainter *painter,
+                            citizen_category *categories, int num_citizens);
```

Comment:
Since the function doesn't need to modify the `categories`, pass them by pointer-to-`const`:
```suggestion
                            const citizen_category *categories, int num_citizens);
```
This prevents accidental modification and may help the compiler optimize.

Reply:
Changed to `const`